11-misc/callback_server.py

```diff
@@ -0,0 +1,42 @@
+from http.server import BaseHTTPRequestHandler, HTTPServer
+import logging
+
+class S(BaseHTTPRequestHandler):
```

Comment:
i know it requires installing something, but I think this would be more readable / easy to follow as a flask server. Maybe we can have setup instructions at the top or in a readme here